udsoncan/client.py

```diff
@@ -2232,3 +2232,353 @@ def send_request(self, request: Request, timeout: int = -1) -> Optional[Response
             return None
 
         return response
+
+    # ====  Authentication Service Client Functions
+    def deauthenticate(self) -> Optional[services.Authentication.InterpretedResponse]:
+        """
+        Sends a deAuthenticate request (sub function of Authentication Service)
+        introduced in 2020 version of ISO-14229-1.
+
+        :Effective configuration: ``exception_on_<type>_response``
+
+        :return: The server response parsed by
+         :meth:`Authentication.interpret_response<udsoncan.services.Authentication.interpret_response>`
+        :rtype: :ref:`Response<Response>`
+        """
+        return self.authentication(
+            authentication_task=services.Authentication.AuthenticationTask.deAuthenticate
+        )
+
+    def verify_certificate_unidirectional(self,
+                                          communication_configuration: int,
+                                          certificate_client: bytes,
+                                          challenge_client: Optional[bytes] = None) -> Optional[
+         services.Authentication.InterpretedResponse]:
+        """
+        Sends a verifyCertificateUnidirectional request (sub function of Authentication Service)
+        :Effective configuration: ``exception_on_<type>_response``
+
+        :param communication_configuration: Configuration information about how to proceed with security
+         in further diagnostic communication after the Authentication (vehicle manufacturer specific).
```

Comment:
Example of indentation issue I get from sphinx.  The line starts with a space. I can silence the error by replacing the space by a tabs (4 spaces)

Reply:
will fix